tests/producer.test.js

```diff
@@ -56,6 +56,24 @@ const Pulsar = require('../index.js');
           batchingEnabled: true,
         })).rejects.toThrow('Failed to create producer: ConnectError');
       });
+
+      test('Producer Name', async () => {
+        const producer = await client.createProducer({
+          topic: 'persistent://public/default/topic',
+        });
+
+        expect(typeof producer.getProducerName()).toBe('string');
```

Comment:
@alanhoff 

Thank you for your contribution.

[nits]
Could you set producerName and check that string?

e.g.
```
        const producer = await client.createProducer({
          topic: 'persistent://public/default/topic',
          producerName: 'aaaaa'
        });

expect(producer.getProducerName()).toBe('aaaaa');
```

Reply:
Fixed